config/src/main/java/com/alibaba/nacos/config/server/service/dump/DumpService.java

```diff
@@ -278,6 +278,25 @@ private Boolean isQuickStart() {
         return isQuickStart;
     }
 
+    private int getRetentionDays() {
+        String val = env.getProperty("retention.days");
```

Comment:
It's better to use property in application.properties. And the property name should be with prefix 'nacos.config'.